ms/MSOper/MSMetaData.cc

```diff
@@ -3571,9 +3913,22 @@ void MSMetaData::_getScanAndSubScanProperties(
     }
     SHARED_PTR<std::map<SubScanKey, SubScanProperties> > myssprops;
     SHARED_PTR<std::map<ScanKey, ScanProperties> > myscanprops;
+#ifdef _OPENMP
+    if (_parallel) {
+        _computeScanAndSubScanPropertiesParallel(
+            myscanprops, myssprops, showProgress
+        );
+    }
+    else {
+        _computeScanAndSubScanProperties(
+            myscanprops, myssprops, showProgress
+        );
+    }
+#else
     _computeScanAndSubScanProperties(
         myscanprops, myssprops, showProgress
     );
+#endif
```

Comment:
and if it is, is the progressmeter worth 200 lines of extra code which is almost a duplicate?
if yes there are certainly ways to restructure it to make the progress meter more useful for tables with few rows.